app/views/content_items/manuals/_manual_layout.html.erb

```diff
@@ -15,8 +19,13 @@
       <% end %>
     <% end %>
 
+    <%= render "govuk_publishing_components/components/heading", {
+      text: I18n.t("manuals.contents_title"),
```

Comment:
Please, see my comment about I18n above.

Also, should a test be written for this one?

Reply:
> Please, see my comment about I18n above.
> 
> Also, should a test be written for this one?



> Please, see my comment about I18n above.
> 
> Also, should a test be written for this one?



> Please, see my comment about I18n above.
> 
> Also, should a test be written for this one?



> Please, see my comment about I18n above.
> 
> Also, should a test be written for this one?

Thanks @unoduetre, I've added a test case for the contents_title and have added that on both hmrc_manual_test.rb & manual_test.rb. Let me know if that's ok?


It looks like the I18n was used previously, not sure if we should leave it just in case if we wanted to add more translation perhaps in the future (looks like it's used elsewhere too). Let me know what you think

Reply:
What I meant is that the part `I18n.` can be removed, leaving only the `t(...)` part. So instead of writing `I18n.t(xyz)` one can write `t(xyz)`.